src/index.ts

```diff
@@ -19,21 +19,21 @@ const reactifyWebComponent = (WC: string) => {
           return undefined;
         }
         if (prop.toLowerCase() === "classname") {
-          return (this.ref.current.className = val);
+          return (this.ref.current.className = val as string);
         }
         if (typeof val === "function" && prop.match(/^on[A-Z]/)) {
           const event = prop[2].toLowerCase() + prop.substr(3);
           this.eventHandlers.push([event, val]);
-          return this.ref.current.addEventListener(event, val);
+          return this.ref.current.addEventListener(event, val as EventListener);
         }
         if (typeof val === "string" || typeof val === "number") {
           this.ref.current[prop] = val;
-          return this.ref.current.setAttribute(prop, val);
+          return this.ref.current.setAttribute(prop, String(val));
```

Comment:
Hey  Vlad, thank you so much for contributing! 

The `setAttribute` method [automatically changes types to String](https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute) if they aren't already. Is this needed for the Types? If so, i would prefer using the TS `as string` so it's compiled away during builds

Reply:
Same with below

Reply:
@BBKolton Thanks for checking this out. Fixed your comments. It is impossible to convert boolean to string using ```as string```. Added ```as unknown``` and then ```as string```